### ISSUE TITLE
Question: Is it possible to document the root

### ISSUE THREAD

**Issue by JAdshead** [ruby-grape/grape-swagger#605]
Hi, 

I am migrating existing endpoints to grape and need to be able to document a call to the root `GET example.com/api`

The endpoint is working correctly but I am unable to get Swagger to document it.
Is it possible to document the root or am I missing something ? 

Thanks,
Jonny

```Ruby
class Api < Grape::API

  desc 'blahh blahh'
  get '/' do
    ....
  end
  
  add_swagger_documentation
end
```

**Comment by LeFnord** [comment]
it seems you found a bug 😉 … feel free to make a PR to fix it

**Comment by LeFnord** [comment]
solved via #616
